static/js/stream_edit.js

```diff
@@ -219,6 +237,16 @@ function show_subscription_settings(sub_row) {
             return item.email;
         },
     });
+    // To check if all realm users are added
+    // Realm members' emails
```

Comment:
Also most of these comments are unnecessary, and usually redundant of what can be explained concisely in a commit message :)